aspnet/migration/rc1-to-rc2.rst

```diff
@@ -28,64 +28,213 @@ You must update the SDK version in ``global.json``, as this file is used to conf
   {
     "projects": [ "src", "test" ],
     "sdk": {
-      "version": "1.0.0-preview1-002702"
+      "version": "1.0.0-preview2-003121"
+    }
+  }
+
+Update Target Framework Monikers (TFMs)
```

Comment:
As I noted in the issue, this is covered by the document here: https://dotnet.github.io/docs/core-concepts/dnx-migration.html. Also, this is not exactly complete since library projects should target `netstandard1.x` and add dependencies to `NETStandard.Library` instead of `Microsoft.NETCore.App` in most cases.